liquibase-core/src/main/java/liquibase/database/OfflineConnection.java

```diff
@@ -47,11 +47,13 @@ public class OfflineConnection implements DatabaseConnection {
     private boolean sendsStringParametersAsUnicode = true;
     private String connectionUserName;
 
+    private static final Pattern PATTERN = Pattern.compile("offline:(\\w+)\\??(.*)");
```

Comment:
Same here, regarding naming. Also, I would follow the same convention of extracting the regex as a separate constant.